src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -91,11 +93,11 @@ public void main() {
         gathererMain();
     }
 
-    public void gathererMain() {
+    public void gathererMain() { // FIXME WON'T RETRACT THE CLIMBER
         if (FeatureFlags.doCompressor && FeatureFlags.compressorInitialized) {
             switch (gathererState) {
                 case gathererUp:
-                    if (oi.manualIntakeButton()) { // Lower intake if button pressed else stop the intakes
+                    if (oi.manualIntakeButtonPress()) { // Lower intake if button pressed else stop the intakes
```

Comment:
If I'm understanding correctly, it seems like this code will only run the intake motor on the first button press and then it will stop. There is a problem with how we are using the button press to raise the gathere also which is why it seems this change was made... we don't currently have a way to detect button release.

Reply:
there actually is a method to detect when the button is released

Reply:
If you look at the state diagram in the todo doc, the transitions are trigger by press and release (not by button being down). If we can detect both types of transitions of the button, this should be a simple fix.